ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -112,6 +124,22 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     }
   }
 
+  const hasETAGroupError = (group: MhrRegistrationHomeOwnerGroupIF): boolean => {
+    // Variable to track if group has a valid combination of Executor/Trustee/Admin (ETA) Owners
+    let hasETAGroupError = false
```

Comment:
We don't really need this variable, as we can simply return the boolean.

Reply:
fixed